internal/admin/config.go

```diff
@@ -23,24 +23,45 @@ import (
 	"net/http"
 
 	"github.com/google/exposure-notifications-server/internal/database"
+	"github.com/google/exposure-notifications-server/internal/secrets"
 	"github.com/google/exposure-notifications-server/internal/setup"
+	"github.com/google/exposure-notifications-server/internal/signing"
+	"github.com/google/exposure-notifications-server/internal/storage"
 )
 
-var _ setup.DBConfigProvider = (*Config)(nil)
+var _ setup.DatabaseConfigProvider = (*Config)(nil)
+var _ setup.KeyManagerConfigProvider = (*Config)(nil)
+var _ setup.SecretManagerConfigProvider = (*Config)(nil)
+var _ setup.BlobstoreConfigProvider = (*Config)(nil)
 
 type Config struct {
+	Database      *database.Config
+	KeyManager    *signing.Config
```

Comment:
is the admin console using these dependencies now?

Reply:
well, I guess implicitly secret manager was being used before.

Reply:
Yea I put Secret Manager everywhere we have a database. I can remove KeyManager, but it doesn't hurt...

Reply:
agreed, doesn't hurt. may be useful in the future even :)